# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../file/elf.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

AVR32 (and formerly Blackfin) matching, just for the fun of it.

  - Rene Rebe <rene@exactcode.de>

--- file-4.20/magic/Magdir/elf.orig	2007-03-02 09:09:25.000000000 +0000
+++ file-4.20/magic/Magdir/elf	2007-03-02 09:18:40.000000000 +0000
@@ -199,2 +198,3 @@
 >>18	beshort		97		NatSemi 32k,
+>>18	beshort		0x18ad		AVR32 (unofficial),
 >>18	beshort		0x9026		Alpha (unofficial),
